app/models.py

```diff
@@ -2038,18 +2034,19 @@ class DateTimeDimension(db.Model):
 class FactNotificationStatus(db.Model):
     __tablename__ = 'ft_notification_status'
 
-    bst_date = db.Column(db.Date, index=True, primary_key=True, nullable=False)
-    template_id = db.Column(UUID(as_uuid=True), primary_key=True, index=True, nullable=False)
+    bst_date = db.Column(db.Date, index=True, primary_key=True, nullable=False, default=datetime.datetime.utcnow)
```

Comment:
This is a [Date field](https://www.postgresql.org/docs/15/datatype-datetime.html). I'm not sure why it accepted a `datetime.datetime` but it should be a `datetime.date` object for the default.